test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaAttributeTests.java

```diff
@@ -28,39 +28,65 @@
 @Test(groups = { "level.sanity" })
 public class ValhallaAttributeTests {
 
-	class MultiPreloadTest {}
+	static class MultiLoadableDescriptorsTest {}
 
-	/* A class may have no more than one Preload attribute. */
-	@Test(expectedExceptions = java.lang.ClassFormatError.class, expectedExceptionsMessageRegExp = ".*Multiple Preload attributes.*")
-	static public void testMultiplePreloadAttributes() throws Throwable {
-		String className = MultiPreloadTest.class.getName();
-		ValhallaAttributeGenerator.generateClassWithTwoPreloadAttributes("MultiPreloadAttributes",
-			new String[]{className}, new String[]{className});
+	/* A class may have no more than one LoadableDescriptors attribute. */
+	@Test(expectedExceptions = java.lang.ClassFormatError.class,
+		expectedExceptionsMessageRegExp = ".*Multiple LoadableDescriptors attributes.*")
+	static public void testMultipleLoadableDescriptorsAttributes() throws Throwable {
+		String className = MultiLoadableDescriptorsTest.class.getName();
+		ValhallaAttributeGenerator.generateClassWithTwoLoadableDescriptorsAttributes(
+			"MultiLoadableDescriptorsAttributes",
+			new String[]{"L" + className + ";"},
+			new String[]{"L" + className + ";"}
+		);
 	}
 
-	class PreloadClass {}
+	static class LoadableDescriptorsClass {}
 
 	@Test
-	static public void testPreloadBehavior() throws Throwable {
-		String className = PreloadClass.class.getName();
-		/* Verify PreloadClass is not loaded */
+	static public void testLoadableDescriptorsBehavior() throws Throwable {
+		String className = LoadableDescriptorsClass.class.getName();
+		/* Verify LoadableDescriptorsClass is not loaded */
```

Comment:
Please finish each sentence with a period (at least those lines added or modified by this change).